frontend/src/app/testResults/uploads/Uploads.test.tsx

```diff
@@ -169,6 +169,19 @@ describe("Uploads", () => {
 
       await new Promise((resolve) => setTimeout(resolve, 0));
       expect(mockIsDone).toEqual(true);
+    });
```

Comment:
very minor nit but it looks like you're splitting this test case into two? I think the description string for the above case needs to be updated to reflect that we're no longer checking if the result is displayed